test/browser_tests/page_objects/page_wagtail_states_pages.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+function DraftPage() {
+
+  this.get = function() {
+    var baseUrl = '/draft-page/';
+    browser.get( baseUrl );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+}
+
+function SharedPage() {
+
+  this.get = function() {
+    var baseUrl = '/shared-page/';
+    browser.get( baseUrl );
+  };
+
+  this.get_staging = function() {
```

Comment:
Minor: Should be `getStaging`
bewds: run `gulp lint` and it'll warn you about style issues.